docs/errors.md

```diff
@@ -0,0 +1,29 @@
+## Errors
+
+`pg_hint_plan` stops parsing on any error and uses hints already parsed on the
+most cases. Following are the typical errors.
+
+#### Syntax errors
```

Comment:
Would it be better to use three '#' here?

Reply:
My understanding is that it's a subclass or the general "Errors" before.  If we change it to be 1 level higher we should move it after "Nested Comments", or before "Errors".